fintech-examples/fintech-ui/angular.json

```diff
@@ -23,10 +23,7 @@
             "polyfills": "src/polyfills.ts",
```

Comment:
Is this reformatting really necessary?

Reply:
automatically done by the pre-commit hook